obspy/clients/filesystem/tsindex.py

```diff
@@ -0,0 +1,1820 @@
+  Returns a list of (network, station, location, channel, earliest, latest)
+  tuples that represent the full extent of available data. This example
+  retrieves from the very small obspy test tsindex database a list of all
+  available ("BHZ") channel extents from the Global Seismograph Network
+  ("IU") for all times.
+
+>>> extents = client.get_availability_extent(network="IU", channel="BHZ")
+>>> for extent in extents:
+...     print("{0:<3} {1:<6} {2:<3} {3:<4} {4} {5}".format(*extent))
+IU  ANMO   10  BHZ  2018-01-01T00:00:00.019500Z 2018-01-01T00:00:59.994536Z
+IU  COLA   10  BHZ  2018-01-01T00:00:00.019500Z 2018-01-01T00:00:59.994538Z
+
+* :meth:`~Client.get_availability()`: Works in the same way as
+  :meth:`~Client.get_availability_extent()` but returns a list of (network,
+  station, location, channel, starttime, endtime) tuples representing
+  contiguous time spans for selected channels and time ranges.
+
+* :meth:`~Client.get_availability_percentage()`:
+  Returns the tuple(float, int) of percentage of available data
+  (``0.0`` to ``1.0``) and number of gaps/overlaps. Availability percentage is
+  calculated relative to the provided ``starttime`` and ``endtime``.
+
+>>> from obspy import UTCDateTime
+>>> avail_percentage = client.get_availability_percentage(
+...     "IU", "ANMO", "10", "BHZ",
+...     UTCDateTime(2018, 1, 1, 0, 0, 0, 19500),
+...     UTCDateTime(2018, 1, 1, 0, 1, 57, 994536))
+>>> print(avail_percentage)
+(0.5083705674817509, 1)
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Requesting Timeseries Data
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* :meth:`~Client.get_waveforms()`:
+  This example illustrates how to request 1 second of available ("IU")
+  timeseries data in the test tsindex database. Results are returned as a
+  :class:`~obspy.core.stream.Stream` object. See the
+  :meth:`~Client.get_waveforms_bulk()`
+  method for information on how to make multiple requests at once.
+
+>>> t = UTCDateTime("2018-01-01T00:00:00.019500")
+>>> st = client.get_waveforms("IU", "*", "*", "BHZ", t, t + 1)
+>>> st.plot()  # doctest: +SKIP
+
+.. plot::
+
+    from obspy import UTCDateTime
+    from obspy.clients.filesystem.tsindex import Client
+    from obspy.clients.filesystem.tests.test_tsindex \
+        import get_test_data_filepath
+    # for this example get the file path to test data
+    filepath = get_test_data_filepath()
+    db_path = os.path.join(filepath, 'timeseries.sqlite')
+    # create a new Client instance
+    client = Client(db_path, datapath_replace=("^", filepath))
+    t = UTCDateTime("2018-01-01T00:00:00.019500")
+    st = client.get_waveforms("IU", "*", "*", "BHZ", t, t + 1)
+    st.plot()
+
+-------------
+Indexer Usage
+-------------
+
+The :class:`~Indexer` provides a high level
+API for indexing a directory tree of miniSEED files using the IRIS
+`mseedindex <https://github.com/iris-edu/mseedindex/>`_ software.
+
+Initialize a indexer object by supplying the root path to data to be indexed.
```

Comment:
a / an ? (not native EN speaker here so I really don't know)

Reply:
should be "an"